patches/xpath.patch

```diff
@@ -0,0 +1,34 @@
+--- ./github.com/jipanyang/gnxi/utils/xpath/xpath.go	2022-10-25 16:16:00.239783329 +0800
++++ ./github.com/jipanyang/gnxi/utils/xpath/xpath.go	2022-10-25 15:56:18.336168598 +0800
+@@ -20,6 +20,7 @@
+ 	"fmt"
+ 	"regexp"
+ 	"strings"
++	"strconv"
+ )
+ 
+ var (
+@@ -135,9 +136,10 @@
+ // For example, given
+ //	"[k1=v1][k2=v2]",
+ // this API returns
+-//	map[string]string{"k1": "v1", "k2": "v2"}.
++//	map[string]string{"[0]k1": "v1", "[1]k2": "v2"}.
```

Comment:
How about compatibility?
If we add num for each key, other GNMI client can't communicate with SONiC telemetry?

Reply:
The scope of this numbering within parameters of a path element.

@pettershao-ragilenetworks  -- please correct my statement if needed.